Automattic/pocket-casts-ios
# Standard Tags: disable episode artwork on cells and rely on `URLCache`

For users with big databases, saving one episode of metadata takes ~1ms. This doesn't look like a big problem, but if you have a huge podcast (such as The Daily) with 2k episodes, this will take ~2s.

All the other queries need to wait for this one to finish, and if you're scrolling through the Filters, for example, this can go exponentially high to the point it's impossible to use the app.

The logic here is changed so we rely on `URLCache` until we're sure we don't have performance issues. It also disabled enabling artwork on episode cells.

## To test

Go to `FeatureFlag.swift` and change `newShowNotesEndpoint`, `episodeFeedArtwork` and `rssChapters` to `true`.

1. Run an instance of the app with a big database 
2. Check this branch and run it ¶
3. Go to a Filter (preferably the one with the most number of episodes)
4. Scroll as fast as you can
5. ✅ You shouldn't experience big freezes
6. Play an episode from a podcast that has artwork (like Clublife)
7. ✅ Episode artwork should appear

¶ Without the debugger attached (you can just close and reopen the app). I experienced that the debug can slow down the app a lot for huge databases.

## Checklist
